akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -467,6 +470,14 @@ object Multipart {
           FastFuture.successful(this)
         override def productPrefix = "FormData.BodyPart.Strict"
       }
+
+      /** Java API */
+      object StrictBuilder {
```

Comment:
Java API can never use default parameters.
One has to manually write the 3 overloads